tests/test_scoring/test_benchmark_definitions.py

```
@@ -25,25 +27,65 @@
-
-    def test_case_insensitive(self):
-        s = ScorerInput(response="answer: d", target="D")
-        assert multichoice_regex(s)["correct"] is True
-
-    def test_no_answer_found(self):
-        s = ScorerInput(response="I'm not sure about this one", target="A")
-        assert multichoice_regex(s)["correct"] is False
-
-    def test_10_choice(self):
+    @pytest.mark.parametrize(
+        "response,target,letters,correct,expected_extracted",
```

Ruff (0.15.18)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)